Makefile

```diff
@@ -12,8 +12,10 @@ for file in /etc/os-release /usr/lib/os-release; do \
     fi \
 done)
 
+HAVEGO=$(shell go version 2>/dev/null)
```

Comment:
This is already used in `golang.mk` so why not use that?

Reply:
Makefile use it to make sure the current work environment can use go command.

Reply:
@jodh-intel Thanks for your remind.
I pushed a new version that replaces HAVEGO to golang_version_raw.